app/views/hyrax/base/_form_warning.html.erb

```diff
@@ -21,8 +21,7 @@
   <li class="radio">
     <label>
```

Comment:
I didn't catch the nested `label`s here. Not sure the solution, maybe individual `label`s with `for='<id>'`?